src/main/webapp/style.css

```diff
@@ -0,0 +1,7 @@
+h1 {
+  font-size: x-large;
+}
+
+.emphasis {
+  color: violet;
+}
```

Comment:
Please look into [Bootstrap](https://getbootstrap.com/) to increase consistency among all of our website. For example, consider using [the display class](https://getbootstrap.com/docs/4.0/content/typography/#display-headings) from Bootstrap instead of defining a larger font size for the header.

Reply:
Will do!